transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -124,18 +129,18 @@ public void testTransformJaxbToDomainAccepted() {
 
         final IIndividualRequest domainObject = updateItemTransformer.transformJaxbToDomain(updateRequestType);
 
-        Assert.assertNotNull(domainObject);
-        Assert.assertEquals("Found correct sdt request id", domainObject.getSdtRequestReference(), sdtRequestId);
-        Assert.assertEquals("Found correct request status", UpdateStatusCodeType.ACCEPTED.value(),
-                domainObject.getRequestStatus());
+        assertNotNull(domainObject);
+        assertEquals(domainObject.getSdtRequestReference(), sdtRequestId, "Found correct sdt request id");

```

Comment:
I think domainObject.getSdtRequestReference() and sdtRequestId are passed to the wrong parameters. The parameters for the assertEquals() method are expect value, actual value and message. It looks like sdtRequestId is the expected value and domainObject.getSdtRequestReference() is the actual value so sdtRequestId should be the first parameter and domainObject.getSdtRequestReference() the second.